### DISCUSSION TITLE
【评论区IP属地显示】这个插件打开了不管用，是不是还有在哪有个开关啥的？

### DISCUSSION THREAD

**Discussion by qaqz111** [the1812/Bilibili-Evolved#4903] [提问 / Q&A] [answered]
确定插件列表上这个插件是被启用了的，研究了半天没找到在是不是还有在哪要打开什么设置。
单独装的那个开盒脚本装上倒是立刻就能显示IP属地了。

**Comment by Geeyun-JY3** [comment, marked as answer]
请
善
用
搜
索
，
看
置
顶
：
#4843

**Reply by qaqz111** [reply]
啊，谢谢，这个置顶我确定是看到过的，但是没太在意忘了，发帖之前只搜了关键字 “IP” 没搜 “评论区”。

**Reply by Geeyun-JY3** [reply]
关键词含「IP」的相关重复 issue 近期也有人开过：#4892、#4897

**Reply by qaqz111** [reply]
是的，但是我没有在 issue 那边搜，只搜了 Dissussions 这边，我一开始觉得就是我的问题没找到开关啥的应该不属于要报 issue 的问题，就没去 issue 那边搜/笑哭

Disscussions 这边搜 IP 有 5 个结果，但是没有说失效问题的，然后才开了贴提问，是我的失误，抱歉给您添麻烦了。
https://github.com/the1812/Bilibili-Evolved/discussions/4506
https://github.com/the1812/Bilibili-Evolved/discussions/4622
https://github.com/the1812/Bilibili-Evolved/discussions/4490
https://github.com/the1812/Bilibili-Evolved/discussions/3522
https://github.com/the1812/Bilibili-Evolved/discussions/3967

**Reply by Geeyun-JY3** [reply]
倒也没给我添什么麻烦，下次注意就好。
